oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexHelper.java

```diff
@@ -52,8 +53,12 @@ class ElasticIndexHelper {
      * Changes not breaking compatibility should increment the minor version (old queries still work, but they might not
      * use the new feature).
      * Changes that do not affect queries should increment the patch version (eg: bug fixes).
+     * <p>
+     * WARN: Since this information might be needed from external tools that don't have a direct dependency on this module, the
+     * actual version needs to be set in oak-search.
      */
-    protected static final String MAPPING_VERSION = "1.1.0";
+    protected static final String MAPPING_VERSION = FulltextIndexConstants.INDEX_VERSION_BY_TYPE.
```

Comment:
Hmm I'm wondering why the default one is `1.0.0` instead of `1.1.0`?

Reply:
`1.1.0` is the current version. It does not make sense to have a default version since the version has to always be defined. It can only be null in case oak-search points to some old version. I have changed the code to handle this case.